docs/pvacbind/additional_commands.rst

```diff
@@ -19,6 +19,18 @@ List Valid Alleles
 
 .. program-output:: pvacbind valid_alleles -h
 
+.. _valid_algorithms:
+
+List Valid Algorithms
+------------------
```

Comment:
The compiler will emit warnings when the title underlines are shorter than the title itself.

```suggestion
List Valid Algorithms
---------------------
```